app/api.js

```diff
@@ -21,7 +20,6 @@ const getConfiguration = (token) => {
     json: true
   }
 }
-
```

Comment:
Was there a reason for removing these spaces?

Reply:
Yes, I have fat fingers.  Restored.